### PR TITLE
fix formatting errors of Roc's part

### PR DESCRIPTION
**Pull request by Zhang-Haipeng**

### CODE REVIEW

File: tests/test_gather_stock_returns.py [hunk 1]
```diff
@@ -6,40 +6,44 @@
...
-    
-def date_format_test():
+
+
+def test_date_format():
     """
     See if the function correctly handles the input dates in wrong formats.
     """
-    msg = "Invalid Date format - please input the date as a string with format %Y-%m-%d"
+    msg = "Invalid Date format - "
```
Comment (slbsolomon) [inline tests/test_gather_stock_returns.py]: Can you edit to make the error message consistent with in the code? This was related to my changes of standardizing error messages last week

**Review by slbsolomon — CHANGES_REQUESTED** [review]
All changes are logical and related to formatting.

**Comment by slbsolomon** [comment]
verified in logs that all of your items are correct now

**Review by sirine-chahma — APPROVED** [review]

File: tests/test_gather_stock_returns.py (hunk 1 above)
Comment (Zhang-Haipeng) [reply tests/test_gather_stock_returns.py]: Fixed, thanks

**Comment by Zhang-Haipeng** [comment]
Please review the updated. Thanks

**Review by slbsolomon — APPROVED** [review]
All changes as discussed - related to error messages and formatting

**Pull request merged**